views/beer.hbs

```diff
@@ -0,0 +1,5 @@
+<div>
+  <div class='beer-container'>
+   {{> beerpartial this}}
```

Comment:
I would use camelCase for that one as it's a little bit more aligned with conventions than a 2 word file and all lower case